server/src/main/kotlin/org/octopusden/octopus/dms/service/impl/ArtifactServiceImpl.kt

```diff
@@ -56,6 +58,9 @@ class ArtifactServiceImpl(
         log.info("Download artifact with ID '$id'")
         val artifact = artifactRepository.findById(id)
             .orElseThrow { NotFoundException("Artifact with ID '$id' is not found") }
+        if (artifact.repositoryType == RepositoryType.DOCKER) {
+            throw UnsupportedOperationException("Docker artifacts can't be downloaded")
```

Comment:
The download of Docker artifacts is not supported.

Reply:
Fixed